### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: src/api/api-alephium.ts [hunk 1]
```diff
@@ -429,12 +429,14 @@ export interface CompileContractResult {
...
   functions: FunctionSig[]
   events: EventSig[]
+  warnings: string[]
 }
 
 export interface CompileScriptResult {
   bytecodeTemplate: string
   fields: FieldsSig
   functions: FunctionSig[]
+  warnings: string[]
```
Comment (Lbqds) [inline src/api/api-alephium.ts:439]: Do we need to print warnings when compiling scripts and contracts?

**Review by Lbqds — APPROVED** [review]
LGTM, only one minor comment.

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: src/api/api-alephium.ts (hunk 1 above)
Comment (polarker) [reply src/api/api-alephium.ts:439]: I forgot to handle warning. I will create another PR. I am not sure if we should print the warnings or we should save it to a file for now.

**Pull request merged**